assets/training/distillation/src/generate_data.py

```diff
@@ -366,14 +365,18 @@ def process_request(idx: str, data: dict, url: str, endpoint_key: str):
             messages = normalize_messages(messages)
             last_status_code = None
             synthetic_responses = []
+            inference_data = []
             for turn_id, message in enumerate(messages):
                 role = message["role"]
                 if role == "system":
-                    synthetic_responses.append(process_system_prompt(message))
+                    # Data for fine-tune job should not include CoT prompt
```

Comment:
@visahan-24 
Can you please check this change.

Reply:
@babu-namburi - This change looks good